### ISSUE TITLE
pointcloud issue

### ISSUE THREAD

**Issue by wearenocomputer** [sadmb/ofxKinect4Azure#2]
Hi I had issues with the point cloud not being correct.

Everything that has to do with depth should be interpreted as uint16 instead of uint8.

uint8_t* d_buffer = depth_image.get_buffer();

should be

uint16_t* d_buffer = depth_image.get_buffer();

This is how I get the correct pointcloud data from a depthmap:

```
//create pointcloud 
k4a::image d_image = capture.get_depth_image();
k4a::image xyz_image = transformation.depth_image_to_point_cloud(d_image, K4A_CALIBRATION_TYPE_DEPTH);
	
int16_t* p_buffer = (int16_t*)xyz_image.get_buffer();

int width = xyz_image.get_width_pixels();
int height = xyz_image.get_height_pixels();

pointcloud_vert.resize(width*height);
		
int counter = 0;
		for (int h = 0; h < height; h++)
		{
			for (int w = 0; w < width; w++)
			{
				int pixelIndex = h * width + w;

				pointcloud_vert[counter].x = p_buffer[3 * pixelIndex + 0] * 0.001f;
				pointcloud_vert[counter].y = p_buffer[3 * pixelIndex + 1] * 0.001f;
				pointcloud_vert[counter].z = p_buffer[3 * pixelIndex + 2] * 0.001f;
				
				counter++;
			}

		}

		pointcloud_vbo.setVertexData(pointcloud_vert.data(), width*height, GL_STREAM_DRAW);
```

**Comment by sadmb** [comment]
@wearenocomputer
Thank you for the contribution.
I changed uint8_t* to short* as dev branch did and it seems vbo works fine.
But it seems color mapping is not working properly yet.
It doesn't work properly also on example_pointcloud of dev branch, so maybe it's caused by api change?
I will try fix it later.
Thanks!

**Comment by sadmb** [comment]
OK, Fixed the problem 3d143ea25bdde7a16651389a51a5ca6673f1965f.
